src/terra-dev-site/contributing/GitWorkflow.b.contributing.md

```diff
@@ -0,0 +1,112 @@
+# Git Workflow
+The follow section outlines our Git workflow for Terra UI projects.
+
+## Fork The Repository
+
+Start by [forking](https://help.github.com/articles/fork-a-repo/) the repo you’ll be working on.
+
+_Note: The [terra team](https://github.com/orgs/cerner/teams/terra) and [terra contributors](https://github.com/orgs/cerner/teams/terra-contributors) can ignore this step and clone the repo directly._
+
+## Clone The Repo
```

Comment:
I feel like the information about cloning the repo and cloning the forked repo can be dropped. This is outlined in the link you included above.

Reply:
Updated: https://github.com/cerner/terra-ui/pull/108/commits/453b149ee491fa9a8d004c85dc550ca83dedf36d